content/darkroom/masking-and-blending/blend-modes.md

```diff
@@ -5,44 +5,159 @@ weight: 20
 draft: false
 ---
 
-Blend modes define how the input and output of a module are combined (blended) together before a module's final output is passed to the next module in the pixelpipe. Most of the blend modes should be familiar from other imaging software and are thoroughly discussed in the [GIMP user manual](https://docs.gimp.org/2.8/en/gimp-concepts-layer-modes.html). 
-
-This section therefore only discusses some of the darktable-specific blend modes in detail.
+Blend modes define how the input and output of a module are combined (blended) together before a module's final output is passed to the next module in the pixelpipe. 
 
 Some blend modes depend on a fulcrum for their operation. This fulcrum usually defines the point at which the blend mode results in a "no-operation" which may be white or grey depending on the blend mode. The additional _blend fulcrum_ parameter allows the position of this fulcrum to be adjusted where blending is performed within the RGB scene-referred color space. The effect depends on the operator used. For example, values above the fulcrum might be brightened and values below darkened, or vice versa.
```

Comment:
I would add here : "Classic blending modes, meant for display-referred RGB (forcefully encoded between 0 and 100 %), implicitly define a fulcrum at 50 % (grey) or 100 % (white) in the code. Because scene-referred is not bounded to 100 % anymore, that fulcrum needs to be explicitely defined by users.".